LICENSE.md

```diff
@@ -23,6 +23,7 @@ The following libraries are included in CKEditor under the [MIT license](https:/
 * color-parse - Copyright (c) 2015 Dmitry Ivanov.
 * emoji-picker-element-data - Copyright (c) 2020 Nolan Lawson.
 * Fuse.js - Copyright (c) 2017 Kirollos Risk.
+* is-emoji-supported - Copyright (c) 2023, Koala Interactive SAS
```

Comment:
The copyright from their `LICENSE.md` is:

```
Copyright (c) 2016-2020 Koala Interactive, Inc.
```

Reply:
@martnpaneq, this entry repeats twice in the PR. 7343da3b2734aaf3d93b6e1e8e455c9a2bf881df includes a single fix only.